plugin/example/source/dextool/plugin/cpptestdouble/runner.d

```diff
@@ -0,0 +1,150 @@
+/**
+Copyright: Copyright (c) 2017, Joakim Brännström. All rights reserved.
+License: MPL-2
+Author: Joakim Brännström (joakim.brannstrom@gmx.com)
+
+This Source Code Form is subject to the terms of the Mozilla Public License,
+v.2.0. If a copy of the MPL was not distributed with this file, You can obtain
+one at http://mozilla.org/MPL/2.0/.
+
+This file contains an example plugin.
+*/
+
+// for the generic main function to find the plugin the module must have this
+// name.
+module dextool.plugin.runner;
+
+import std.stdio;
+import logger = std.experimental.logger;
+
+import dextool.type : ExitStatusType;
+
+// called by the generic main function in standard.d
+ExitStatusType runPlugin(string[] args) {
```

Comment:
Again it's unclear what to expect that the plugin will do with the ``string[] args`` here if we don't know that the goal of the plugin is.